packages/in_app_purchase/test/store_kit_wrappers/sk_methodchannel_apis_test.dart

```diff
@@ -133,6 +133,13 @@ void main() {
       expect(fakeIOSPlatform.applicationNameHasTransactionRestored, 'aUserID');
     });
   });
+
+  group('Code Redemption Sheet', () {
+    test('presentCodeRedemptionSheet should not throw', () async {
+      await SKPaymentQueueWrapper().presentCodeRedemptionSheet();
+      expect(fakeIOSPlatform.presentCodeRedemption, true);
+    });
```

Comment:
Check `fakeIOSPlatform.presentCodeRedemption` is false before the test, and reset it to false after.

```
expect(fakeIOSPlatform.presentCodeRedemption, false);
await SKPaymentQueueWrapper().presentCodeRedemptionSheet();
expect(fakeIOSPlatform.presentCodeRedemption, true);
fakeIOSPlatform.presentCodeRedemption = false;

```
Ideally, we should have added a `reset` method to reset all the test params in `FakeIOSPlatform` and run in on `setup`, but we can do that later.